blockchain_geth/src/lib.rs

```diff
@@ -1,6 +1,133 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
+extern crate serde_json;
+
+use std::process::{Command};
+use serde_json::Value;
+
+fn code(tx_to: &str, tx_from: &str, blk_no: &str, address: &str) {
+    let prefix = "{\"jsonrpc\":\"2.0\",\"method\":\"eth_getCode\",\"params\":[\"";
```

Comment:
Most of this code is undergoing change, but also it's best to try to avoid upfront consensus ;-) You're completely welcome to make a PR, but if you don't it just means it's not a big enough problem.